WordPress/Classes/ViewRelated/Site Creation/Shared/TableViewOffsetCoordinator.swift

```diff
@@ -15,20 +15,37 @@ final class TableViewOffsetCoordinator {
     /// The table view to coordinate
     private weak var tableView: UITableView?
 
+    //// The view containing the toolbar
+    private weak var container: UIView?
+
+    //// The toolbar
+    private weak var toolbar: UIView?
+
+    //// The constraint linking the bottom of the toolbar to its container
+    private weak var toolbarBottomConstraint: NSLayoutConstraint?
+
     /// The value of the bottom constraint constant is set in response to the keyboard appearance
     private var keyboardContentOffset = CGFloat(0)
 
     /// To avoid wasted animations, we track whether or not we have already adjusted the table view
     private var tableViewHasBeenAdjusted = false
 
+    private var toolbarHasBeenAdjusted = false
```

Comment:
I have two questions about this property:

1. Would you mind adding a comment for it?
2. Is it ever set back to `false`? Does it need to be?

Reply:
Both fair questions!